libzeth/circuits/joinsplit.tcc

```diff
@@ -74,7 +74,7 @@ private:
     std::array<std::shared_ptr<libsnark::digest_variable<FieldT>>, NumInputs>
         input_nullifiers;
     // List of commitments generated for the new notes
-    std::array<std::shared_ptr<libsnark::digest_variable<FieldT>>, NumOutputs>
+    std::array<std::shared_ptr<libsnark::pb_variable<FieldT>>, NumOutputs>
```

Comment:
Can't we use a `pb_variable_array` here?

Reply:
Otherwise, why not switch to an array of `pb_variable`, i.e. `std::array<pb_variable , NumOutputs>`?

Reply:
Updated.